### ISSUE TITLE
width and height adjustment

### ISSUE THREAD

**Issue by annuvin** [espterm/espterm-firmware#265]
it won't allow me to select more than 65 * 30 for the windows size, i need at least 80 * 40.

**Comment by MightyPork** [comment]
Yes, if I remember correctly, there's a maximal number of cells you can have. It's given by the RAM size of the ESP. You might be able to increase the limit slightly, but I doubt 80x40 is possible. The standard should be 80x25.
